projects/canopy/src/lib/promo-card/promo-card.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { LgPromoCardComponent } from './promo-card/promo-card.component';
+import { LgButtonModule } from '../button';
```

Comment:
I'm not sure if these need `.module` adding or not, maybe this works now.

Reply:
I think IDEs tend to omit the `.module`, wonder if we should recommend a config until `ng-packagr` fixes it